tensorboard/backend/application.py

```diff
@@ -337,6 +342,55 @@ def _create_wsgi_app(self):
     app = _handling_errors(app)
     return app
 
+  @wrappers.Request.application
+  def _serve_plugin_entry(self, request):
+    """Serves a HTML for iframed plugin entry point.
+
+    Args:
+      request: The werkzeug.Request object.
+
+    Returns:
+      A werkzeug.Response object.
+    """
+    name = request.args.get('name')
```

Comment:
Curious whether you considered `/data/plugin_entry/PLUGIN` or similar
rather than a query parameter? No strong preference.

Reply:
As discussed, since we can change this transparently, I will keep it as is. If we have better support in the future for url parameters like `/foo/bar/:baz`, we will definitely consider doing that.

Reply:
SGTM; thanks!